### PR TITLE
fix(SUP-37810):V7 multi-source entries in Safari

### PR DESCRIPTION
**Pull request by inbalvasserman**
the issue:
on Safari when run dual screen the video is getting stuck all the time. 

solution:
add sync delay threshold for Safari only to reduce the number of times it change the second player rate

### CODE REVIEW

File: src/video-sync-manager.ts [hunk 1]
```diff
@@ -117,16 +117,21 @@ export class VideoSyncManager {
     const synchDelayThresholdNegative = -0.03;
     const maxGap = 4;
     const seekAhead = 0.25; // s
+    //on safari without it the video get stuck on every rate change
+    // @ts-ignore
+    const synchDelayDefaultThreshold = this._secondaryPlayer._localPlayer._env.browser.name == "Safari" ? 0.5 : 0;
```
Comment (semarche-kaltura) [inline src/video-sync-manager.ts:122]: KalturaPlayer.core.Env.isSafari

Comment (semarche-kaltura) [inline src/video-sync-manager.ts:122]: why value is exact 0.5?

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

File: src/video-sync-manager.ts (hunk 1 above)
Comment (inbalvasserman) [reply src/video-sync-manager.ts:122]: this is the lowest value that I found that fix the issue. with less then 0.5  there are still lots of problems in the video

File: src/video-sync-manager.ts [hunk 2]
```diff
@@ -117,16 +117,21 @@ export class VideoSyncManager {
     const synchDelayThresholdNegative = -0.03;
     const maxGap = 4;
     const seekAhead = 0.25; // s
+    //on safari without it the video get stuck on every rate change
+    // @ts-ignore
+    const synchDelayDefaultThreshold = this._secondaryPlayer._localPlayer.env.isSafari ? 0.5 : 0;
```
Comment (semarche-kaltura) [inline src/video-sync-manager.ts:122]: could you add comments why we use 0.5 value for future maintenance

**Review by semarche-kaltura — APPROVED** [review]

**Pull request merged**